examples/function/asyncio-get/async_exec.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+import json
+from typing import AsyncIterable
+
+import aiorun
+import aiohttp
+from pynumaflow import setup_logging
+
+from pynumaflow.function import Messages, Message, Datum, Metadata, UserDefinedFunctionServicer
+
+_LOGGER = setup_logging(__name__)
+
+
+async def http_request(session, url):
+    async with session.get(url) as resp:
+        result = await resp.read()
+        try:
+            res_json = json.loads(result)
+            return res_json['message']
+        except Exception as e:
+            _LOGGER.error("HTTP request error: %s", e)
+            return "Error"
+
+
+async def reduce_handler(key: str, datums: AsyncIterable[Datum], md: Metadata) -> Messages:
+    interval_window = md.interval_window
+    async with aiohttp.ClientSession() as session:
+        tasks = []
+        async for _ in datums:
+            url = f'http://host.docker.internal:9888/ping'
+            tasks.append(asyncio.ensure_future(http_request(session, url)))
```

Comment:
we dont need ensure_future in this case since http_request is already an async coroutine.

Reply:
Done.